kodilla-stream/src/main/java/com/kodilla/stream/forumuser/Forum.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.stream.forumuser;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Forum {
+    private final List<ForumUser> forumUsers;
+    private final int numbersOfUsers;
+
+    public Forum(int numbersOfUsers) {
+        this.numbersOfUsers = numbersOfUsers;
+        this.forumUsers = generateForumUsers();
+    }
+
+    public List<ForumUser> getForumUsers() {
+        return new ArrayList<>(this.forumUsers);
+    }
+
+    private List<ForumUser> generateForumUsers() {
```

Comment:
to trzeba gdzies na zewnatrz, forum nie powinno generowac swoich userow, powinno dostac ich z zewnatrz